providers/onshape/onshape_provider/edge.py

```diff
@@ -89,3 +89,34 @@ def set_is_construction(self, is_construction: bool):
 
     def get_is_construction(self) -> bool:
         raise NotImplementedError()
+
+
+
+"""
```

Comment:
Could you put this in a json file in a `stubs` folder in the `tests/stubs` directory instead of here, please? Thank you.

Reply:
Okay. And I will replace it with like below
`"""
sample native_instance data stored on /tests/stubs driectory
"""`

Reply:
Sorry, I am firm on moving this sample json out of the file. Having it here means someone will have to update it if the Onshape API changes, and this is not the place where we need this comment. It can live in a stubs folder in tests and a test can be written to ensure that the response schema is valid and what we expect, but otherwise it provides no use in this file. Thank you for your understanding, @buckineer.